### PR TITLE
Add shared SDK cucumber tests to PHP repo

### PR DESCRIPTION
**Pull request by jeskew**

### CODE REVIEW

File: tests/Integ/SmokeContext.php [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+    Context,
+    SnippetAcceptingContext
+{
+    protected static $configDefaults = [
+        'version' => 'latest',
+        'region' => 'us-east-1',
+        'profile' => 'integ',
+    ];
+
+    protected static $services = [
```
Comment (jeremeamia) [inline tests/Integ/SmokeContext.php]: You could use the [manifest file](https://github.com/aws/aws-sdk-php/blob/master/src/data/manifest.json) for this data.

**Comment by jeremeamia** [comment]
Looks great aside from my one comment. Nice work on this. Thank you.

File: tests/Integ/SmokeContext.php (hunk 1 above)
Comment (jeskew) [inline tests/Integ/SmokeContext.php]: The tags on the feature files aren't 1-to-1 matches to what's in the manifest file. I removed the exact matches from the latest commit but there's still a sizable config array.

Comment (jeremeamia) [inline tests/Integ/SmokeContext.php]: Gotcha. I'm working on something that will allow you to remove the rest of that array except the EFS region.

Comment (jeskew) [inline tests/Integ/SmokeContext.php]: I'll hold on this PR until #641 is in master.

Comment (jeskew) [inline tests/Integ/SmokeContext.php]: `\Aws\manifest` worked for everything except ElasticLoadBalancing, so I just changed the tag on that feature.

**Comment by jeremeamia** [comment]
What are we waiting on for this PR?

**Pull request merged**